src/asmcnc/keyboard/custom_keyboard.py

```diff
@@ -12,6 +12,7 @@
 from kivy.core.image import Image
 from kivy.graphics import Color, BorderImage
 from kivy.utils import rgba
+from itertools import cycle
```

Comment:
This was imported but can't find a usage from this pr

Reply:
I actually made a mistake taking it out of the last PR, because I thought it wasn't needed. Then realised, popped it back in this one :)